disasm-test/Main.hs

```diff
@@ -592,7 +602,8 @@ assembleToBitCode pfx file = do
 compileToBitCode :: FilePath -> FilePath -> TestM FilePath
 compileToBitCode pfx file = do
   tmp <- liftIO getTemporaryDirectory
-  Clang comp <- gets clang
+  Clang comp' <- gets clang
+  let comp = if ".cc" `isSuffixOf` file then comp' <> "++" else comp'
```

Comment:
There are some other language extensions that we could consider checking for, such as `.cpp`, `.cxx`, and `.C`. (List taken from [here](https://github.com/GaloisInc/crucible/blob/2cf85d801d806a5f62cfc19403e041169be66cb7/crux-llvm/src/Crux/LLVM/Compile.hs#L48-L50).)